### DISCUSSION TITLE
Menu won’t open on mobile — just me?

### DISCUSSION THREAD

**Discussion by robacarp** [AllYourBot/hostedgpt#220] [General]
I’m on iOS / Safari, and I can’t seem to open the menu. The tooltip shows, but the menu hamburger does nothing.

**Comment by keithschacht** [comment]
@robacarp hmm, is your branch sync'd with the latest? I just tried mine on iOS safari and it’s working. This is how it should look:

Uploading RPReplay_Final1711074104.mp4…

**Reply by keithschacht** [reply]
https://github.com/allyourbot/hostedgpt/assets/35061/d2fcf540-65f2-4ddf-9130-2b5a5ad7c52c

**Reply by keithschacht** [reply]
I’m thinking of what the differences might me:

I just tried opening Safari and I tried bookmarking to home screen and launching from there. For me it worked on both.

I also just tried it on iOS iPad and that worked too (I used split screen in order to get it to collapse).

**Reply by robacarp** [reply]
Ah yeah it is working now sorry. I didnt realize it was out of date

**Reply by keithschacht** [reply]
@robacarp You know what’s cool, I figured out how to make github fork’s auto-update! Now that you’ve updated to the latest it should do a highly sync at ~3am or so.

I don’t fully understand the limitations of this yet. For example, I think it may stop eventually after some inactivity period, which I don’t fully understand. But anyway, it should help things, going forward.

**Reply by robacarp** [reply]
Wonderful! Glad to hear it.
